### PR TITLE
Add credit eligibility information for course authors

### PR DESCRIPTION
**Pull request by srpearce**
@catong - This is the new branch for course author info about credit eligibility (including ICRV and proctoring).

### CODE REVIEW

**Comment by srpearce** [comment]
@wedaly, @griffresch, @lou-wang, @jhendersonedx - This PR has the latest documentation for ICRV. You can see a built version [here](http://draft-credit-eligibility.readthedocs.org/en/latest/building_course/credit_courses/in_course_reverification.html); it doesn't have screen shots yet, but it does have instructions. Feel free to share this with course teams if they want to see docs before I return next Tuesday.

@wedaly, can you do a tech review?

@mhoeber, @lamagnifica, or @catong, feel free to do an editing pass. I've indicated where the screen shots will go.

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Specify the Minimum Credit-Eligible Grade
+********************************************
+
+In addition to setting the usual grade scale for your course, you indicate a
+specific grade that learners must receive to earn credit for your course.
+
+#. In Studio, select **Settings**, and then select **Grading**.
+#. In the **Credit Eligibility** section, specify a grade followed by
+   a percent sign (%) in the **Minimum Credit-Eligible Grade** box.
+   This grade can be any number from 1 to 100.
```
Comment (wedaly) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst]: There's a restriction in place right now that the minimum grade must be >= the passing grade for the course.

Comment (wedaly) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst]: The % sign has been moved out of the text input, so the user just needs to enter the number.

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+.. _Modify Settings:
+
+================================================
+Modify Component and Unit Settings
+================================================
+
+After you create a verification checkpoint, that checkpoint appears in the
+settings for all components and units in the course.
+
+.. add image when sandbox ready (8/19: waiting for new sandbox)
```
Comment (wedaly) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: http://studio-wedaly.sandbox.edx.org/course/course-v1:edx+icrv-access+2015

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+.. note:: If you do not change the settings for a unit, that unit is 
+ visible before the learner passes the checkpoint.
+
+#. On the **Course Outline** page, locate the unit that you want, and then
+   select **Configure** for that unit. The **Configure** control resembles a
+   gear.
+
+#. Under **Verification Checkpoint**, select the name of the verification
+   checkpoint that you want.
```
Comment (wedaly) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:165]: Maybe worth adding a warning here about gating the ICRV block itself, or its parent unit.  If this happens, users wouldn't see the ICRV block, so they'd never be able to submit photos and get access to the exam.

I know this is terrible UI -- there are technical limitations that prevent us from making this more foolproof :)

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+******************
+The Learner View
+******************
+ 
+Learners who must complete in-course reverification see the verification
+checkpoints for their course in the list of course credit requirements on
+their **Progress** pages.
+
+.. update image when sandbox ready (8/19: waiting for new sandbox)
```
Comment (wedaly) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: http://wedaly.sandbox.edx.org/courses/course-v1:edx+icrv-access+2015/info

Comment (wedaly) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: The test user `verified@example.com` is enrolled as verified; the user `honor@example.com` is enrolled as honor.

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+If your institution has an agreement with edX, you might be able to offer
+academic credit for your course. Learners will be able to purchase this
+academic credit after they meet requirements that you set. These requirements
+typically include periodic identity checks throughout the course as well as ID
+verification at the beginning of the course. Learners can see a list of the
+requirements on their **Progress** pages.
+
+If your course offers academic credit, your edX program manager will
+complete the initial setup of your course, including setting the price of
+credit and enabling your course to offer credit. You must then complete two
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst]: two --> these (cause specifying a count in docs always gets you in the end)

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+   a percent sign (%) in the **Minimum Credit-Eligible Grade** box.
+   This grade can be any number from 1 to 100.
+
+.. _Add Identity Checks:
+
+****************************
+Add Identity Checks
+****************************
+
+For courses that offer academic credit, edX has two forms of identity checks:
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst:54]: has two --> offers the following

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst [hunk 7]
```diff
@@ -0,0 +1,56 @@
+.. _Academic Course Credit:
+
+#####################################
+Academic Course Credit
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst:4]: ^Offering Academic...
(topic headings start with a gerund unless for a procedure, which start with an infinitive)

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+#. In Studio, select **Settings**, and then select **Grading**.
+#. In the **Credit Eligibility** section, specify a grade followed by
+   a percent sign (%) in the **Minimum Credit-Eligible Grade** box.
+   This grade can be any number from 1 to 100.
+
+.. _Add Identity Checks:
+
+****************************
+Add Identity Checks
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst:51]: Adding Identity...

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 9]
```diff
@@ -0,0 +1,157 @@
+.. _In Course Reverification:
+
+#################################
+In-Course Reverification
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: ^Using In-Course...

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+#################################
+
+To use in-course reverification, you must enable reverification in your
+course, and then create specific verification checkpoints. Learners who must
+complete identity reverification see these checkpoints on their **Progress**
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: is it identity or in-course reverification? or, if they're different, maybe introduce each one to differentiate them

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+
+#. At the bottom of the page, select **Save Changes**.
+
+.. _Create ICRV Checkpoint:
+
+*************************************************
+Create an In-Course Verification Checkpoint
+*************************************************
+
+Creating an in-course verification checkpoint has two steps.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: two --> the following

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+   Component>`.
+#. :ref:`Modify the settings of the components and units <Modify Settings>`
+   that are behind the checkpoint.
+
+ .. _Create Checkpoint Component:
+
+================================================
+Create a Verification Checkpoint Component
+================================================
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:52]: maybe add an introductory sentence that describes the relationship between the assessment mentioned on the next line and the checkpoint

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+#. In the subsection that contains the assessment, create a new unit.
+
+   If the subsection already contains one or more units, move the new unit so
+   that it is the first unit in the subsection.
+
+#. In the new unit, select **Advanced** under **Add New Component**, select
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: there's a lot to do in this one step... (!) 
a slight reorg will walk readers though what they need to look for in the UI in the order they need to do things in:
In the new unit, under **Add New Component** select **Advanced**...

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+   **Verification Checkpoint**, and then select **Edit**.
+
+#. In the **Verification Checkpoint Name** box, enter a name for the
+   verification checkpoint.
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:63]: maybe, 
Enter a **Verification Checkpoint Name**.

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+#. In the **Verification Attempts** box, select the number of times that
+   learners can attempt to verify their identities. You can specify up to
+   three attempts.
+
+The unit that contains the verification checkpoint component can contain other
+components. If the unit contains other components, edX recommends that you
+position the verification checkpoint as the first component in the unit.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: this seems like info that would be helpful to know before you start the process -- move to above the procedural steps?

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: @wedaly this link hangs for me...

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+
+.. _Modify Settings:
+
+================================================
+Modify Component and Unit Settings
+================================================
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:101]: I'm not really grasping what is going on here. After you add the ICRV requirement in a unit, do you have to go in to every individual unit in the entire course (including, potentially, units that you add after you add the ICRV) and decide whether to change a flag to make the content visible to learners who have self-identified as wanting credit? Or is it only to other units in the same subsection, or other components in the same unit? (compare "all components and units in the course" on line 83 to lines 102-103). 
If you're making this kind of show it/don't show it decision, maybe rather than "that checkpoint appears in the settings" something a little more active, like, "a new setting is available to give you control over the visibility of course components and units."

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+.. image  
+..  :width: 500
+..  :alt: Image of the settings editor for a component and for a unit
+
+To prevent learners from seeing components and units before they pass the
+verification checkpoint, you must select that verification checkpoint in the
+settings for each component and unit in the subsection that you do not want
+learners to see.
+
+Change Component Settings
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: maybe make this more specific to what needs to be accomplished, like Set Component Visibility for Unverified Learners

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+
+.. note:: If you do not change the settings for a component, that component 
+ is visible before the learner passes the checkpoint.
+
+If the unit that contains the verification checkpoint contains one or more
+additional components, follow these steps for each component.
+
+#. On the unit page, locate the component that you want, and then select
+   **Visibility Settings** for that component. The **Visibility Settings**
+   control resembles an eye.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:135]: is it an icon? I think we quote descriptions of icons, but I may be confused

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+
+#. Under **Make visible to**, select **Specific Content Groups**, and then
+   select the verification checkpoint that you want.
+
+Change Unit Settings
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: ditto on making this more descriptive

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 20]
```diff
@@ -0,0 +1,157 @@
...
+#. On the **Course Outline** page, locate the unit that you want, and then
+   select **Configure** for that unit. The **Configure** control resembles a
+   gear.
+
+#. Under **Verification Checkpoint**, select the name of the verification
+   checkpoint that you want.
+
+
+******************
+The Learner View
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: start with a verb

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 21]
```diff
@@ -0,0 +1,157 @@
...
+.. update image when sandbox ready (8/19: waiting for new sandbox)
+
+.. image
+.. ../Images/SFD_Credit_ReqList.png
+.. :width: 500
+.. :alt: Learner's Progress page with a list of credit requirements below the
+..     progress graph.
+
+When learners arrive at a checkpoint in the courseware, learners receive a
+message that lets them know that they must complete identity verification, and
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: second learners --> they

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 22]
```diff
@@ -0,0 +1,157 @@
...
+
+.. add image when sandbox ready (8/19: waiting for new sandbox)
+
+.. image
+.. ../Images/
+.. :width: 500
+.. :alt: A verification checkpoint in the course.
+
+When learners who do not have to complete in-course reverification arrive at a
+checkpoint, the learners receive a message about identity verification and
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: the learners --> they

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 23]
```diff
@@ -0,0 +1,157 @@
...
+verified certificates. They can access assessment content.
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: "They can access assessment content." assumes that the checkpoint is immediately before a problem (I think) which maybe isn't required?

File: en_us/course_authors/source/front_matter/change_log.rst [hunk 24]
```diff
@@ -145,7 +129,7 @@ May 2015
        is released.
    * - 5 May 2015
      - Updated the :ref:`Student Data` section to include information about
-       the learner engagement report.
+       the :ref:`learner engagement report<Track Student Activity>`.
```
Comment (lamagnifica) [inline en_us/course_authors/source/front_matter/change_log.rst]: maybe take the ref off of Student Data then?

**Comment by lamagnifica** [comment]
I took a pass through. Some areas are a little hard to follow right now, perhaps just need more conceptual info or more clearly defined prerequisites.

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst (hunk 2 above)
Comment (wedaly) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: Hmm, it's working for me.  Maybe try again?

Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: all better, thanks

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst (hunk 18 above)
Comment (srpearce) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:135]: We only use descriptions of icons when the icon doesn't have hover text.

**Comment by srpearce** [comment]
@lamagnifica I've been wrestling with this all day and can't brain it anymore. There's so much to explain and it's hard to do it elegantly. Can you take another look at the reworked in_course_reverification.rst file?

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 25]
```diff
@@ -0,0 +1,204 @@
+.. _In Course Reverification:
+
+#################################
+In-Course Identity Reverification
+#################################
+
+To use in-course identity reverification, you must enable reverification in
+your course, and create specific verification checkpoints, and then modify the settings for the components and units that you want to hide behind each checkpoint.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: strike first "and" on this line

**Comment by srpearce** [comment]
@lamagnifica Sorry, hold on, I'm working on it a bit more...

**Comment by srpearce** [comment]
@lamagnifica All right, this time for realsies. Any suggestions are helpful.

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst [hunk 26]
```diff
@@ -0,0 +1,56 @@
...
+****************************
+
+For courses that offer academic credit, edX offers the following identity
+checks: in-course identity reverification and proctored exams. You can use either
+feature or both features in your course.
+
+* In-course identity reverification allows you to add identity verification
+  "checkpoints" at specific locations in your course, such as before
+  assessments or exams. When learners reach a checkpoint, they must use the
+  webcams on their computers to verify their identity, similar to the original
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst:61]: identity  --> identities (for agreement, though this is getting somewhat awkward)

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst [hunk 27]
```diff
@@ -0,0 +1,56 @@
...
+********************************************
+
+In addition to setting the usual grade scale for your course, you indicate a
+specific grade that learners must receive to earn credit for your course.
+
+#. In Studio, select **Settings**, and then select **Grading**.
+#. In the **Credit Eligibility** section, specify a grade in the **Minimum
+   Credit-Eligible Grade** box. This grade must be greater than or equal to
+   the passing grade for the course.
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst:44]: xref to grading (and maybe that section could xref to here?)

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst (hunk 8 above)
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst:51]: ah, I see. This isn't actually a procedure, it's a concept that introduces the types of identity checks that are available. Hmmm. 
and from here, you go on to another section, with concepts and multiple procedures.
I think maybe  right now lines 16-21 don't really set the reader up for the actual work they'll have to do. Maybe instead of a two-step process, there's a prerequisite (set the minimum grade for credit) followed by choosing where in the course and what type of check to do?

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 28]
```diff
@@ -0,0 +1,225 @@
...
+*****************************************
+Enable In-Course Identity Reverification
+*****************************************
+
+To enable in-course identity reverification for your course, follow these
+steps.
+
+#. In Studio, select **Settings**, and then select **Advanced Settings**.
+#. In the **Advanced Module List** field, place your cursor between the
+   brackets (``[]``),and then add ``"edx-reverification-block"``. Make sure to
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: missed space after (`[]`),

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 29]
```diff
@@ -0,0 +1,225 @@
...
+
+#. At the bottom of the page, select **Save Changes**.
+
+.. _Create ICRV Checkpoint:
+
+*********************************
+Create a Verification Checkpoint
+*********************************
+
+To require learners to verify their identity before a specific assessment, you
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: identities

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 30]
```diff
@@ -0,0 +1,225 @@
...
+add a verification checkpoint component to a unit in that assessment. After
+you create a checkpoint component, the checkpoint appears on the **Progress**
+pages and in the courseware for learners who must complete in-course identity
+reverification.
+
+.. Not sure about leaving this note in
+
+.. note:: Although it is possible to add a verification checkpoint at any
+ location, edX strongly recommends that you add verification checkpoints only
+ before assessments.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: maybe simpler would be stronger?
Note: Make sure that you only add verification checkpoints before assessments.

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 31]
```diff
@@ -0,0 +1,225 @@
...
+
+#. In the subsection that contains the assessment, create a new unit.
+
+   If the subsection already contains one or more units, move the new unit so
+   that it is the first unit in the subsection.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:56]: yes!

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 32]
```diff
@@ -0,0 +1,225 @@
...
+
+#. Select **Verification Checkpoint**, and then select **Edit**.
+
+#. In the **Verification Checkpoint Name** box, enter a name for the
+   verification checkpoint.
+
+#. In the **Verification Attempts** box, select the number of times that
+   learners can attempt to verify their identities. You can specify up to
+   three attempts.
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:67]: is there a save or other nav at this point?

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 33]
```diff
@@ -0,0 +1,225 @@
...
+   If the subsection already contains one or more units, move the new unit so
+   that it is the first unit in the subsection.
+
+#. In the new unit, locate **Add New Component**, and then select **Advanced**. 
+
+#. Select **Verification Checkpoint**, and then select **Edit**.
+
+#. In the **Verification Checkpoint Name** box, enter a name for the
+   verification checkpoint.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: Enter an identifying **Verification Checkpoint Name**.

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst (hunk 32 above)
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:67]: ok, when reading lines 83-86, it might be helpful to know at this point that the result of this procedure  (I'm guessing a bit here) is that this checkpoint is now available for selection in the settings for all of the components in the (unit? subsection?).
with that sort of lead-in info, the first paragraph in the next section could be softened somewhat: instead of "you must set" it could be "you can now specify which"

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 34]
```diff
@@ -0,0 +1,225 @@
...
+  checkpoint, all of the components in that unit are invisible. You cannot
+  make an individual component visible.
+
+* If you do not set a unit to be invisible, all of the components in that unit
+  are visible by default. However, you can set individual components to be
+  invisible until the learner passes the verification checkpoint.
+
+.. important:: Do not set the unit that contains the verification 
+ checkpoint component, or the verification component itself, to require
+ verification. If you do this, learners cannot access the verification
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:112]: I prefer this "require verification" language to the visible/invisible designation used elsewhere (I think that they mean the same thing, right?)

oh, I just went backed and looked at the image -- looks like "visibility" is what the UI gives you to work with... maybe instead of visible/invisible go toward "visibility after verification" or something like that? "invisible" just has so many casual connotations....

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 35]
```diff
@@ -0,0 +1,225 @@
...
+the checkpoint that you created. You can also leave some components or units
+visible.
+
+To make a component or unit invisible, you select the verification checkpoint
+in the settings of the component or unit. By default, all components and units
+are visible. To leave them visible, do not select a verification checkpoint in
+the component or unit settings.
+
+Before you make a component or unit invisible, become familiar with the
+following information.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: maybe "become familiar with" --> "note"?

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 36]
```diff
@@ -0,0 +1,225 @@
...
+   control resembles an eye.
+
+#. Under **Make visible to**, select **Specific Content Groups**, and then
+   select the verification checkpoint that you want.
+
+   .. image:: ../Images/ICRV_CompSettings.png
+    :width: 250
+    :alt: The settings editor for a component, showing the list of
+      verification checkpoints.
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:144]: is there a save or other closing step?

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 37]
```diff
@@ -0,0 +1,225 @@
...
+   gear.
+
+#. Under **Verification Checkpoint**, select the name of the verification
+   checkpoint that you want.
+
+   .. image:: ../Images/ICRV_UnitSettings.png
+    :width: 250
+    :alt: The settings editor for a unit, showing the list of verification
+      checkpoints.
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst:171]: ditto

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 38]
```diff
@@ -0,0 +1,225 @@
...
+.. _Verification Example:
+
+=================================================
+See an Identity Verification Example
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: See an? I'm not sure how to read this title

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 39]
```diff
@@ -0,0 +1,225 @@
...
+message that lets them know that they must complete identity verification, and
+that if they do not complete identity verification, they will not be eligible
+for course credit.
+
+.. image:: ../Images/ICRV_Checkpoint.png
+ :width: 500
+ :alt: A verification checkpoint showing a message that the learner must
+     perform identity verification.
+
+After learners pass the checkpoint, they
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: ?

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 40]
```diff
@@ -0,0 +1,225 @@
...
+After learners pass the checkpoint, they 
+
+.. image:: ../Images/ICRV_Checkpoint_Passed.png
+ :width: 500
+ :alt: A verification checkpoint showing a message that the learner can
+     continue to the assessment.
+
+When learners who do not have to complete in-course identity reverification
+arrive at a checkpoint, the learners receive a message about identity
+verification and verified certificates. They can access assessment content.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: they do? ugh

**Comment by lamagnifica** [comment]
much clearer, thanks for working through this again @srpearce  . I made some transition/flow type comments, which might/not be helpful for taking a step back and smoothing out the last bits

**Comment by srpearce** [comment]
OK, I think this is just about done. @wedaly, can you give this a final TR? @lamagnifica, can you look through from a docs standpoint?

(Adding the new doc team PR template below)  

### HTML version

http://draft-credit-eligibility.readthedocs.org/en/latest/building_course/credit_courses/index.html

### Sandbox
- [ ] http://studio.wedaly.sandbox.edx.org / http://wedaly.sandbox.edx.org

### Reviewers

Please review for accuracy and completeness.
- [ ] Subject matter expert: @wedaly 
- [ ] Doc team review: @lamagnifica 
- [ ] PM review: @jhendersonedx

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 41]
```diff
@@ -0,0 +1,246 @@
...
+
+#. In the **Verification Checkpoint Name** box, enter a name.
+
+#. In the **Verification Attempts** box, select the number of times that
+   learners can attempt to verify their identities. You can specify up to
+   three attempts.
+
+#. Select **Save**.
+
+After you save the component, the verification checkpoint name appears in a
```
Comment (wedaly) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: The component also needs to be published before it will show up in component/unit settings.

**Comment by srpearce** [comment]
Ready to merge when we get the go-ahead. For now, academic credit documentation is draft-only.

**Comment by srpearce** [comment]
Per meeting with @mhoeber and @catong: Carol will merge proctoring documentation separately because it needs to go out now. She'll comment out any xrefs to credit eligibility/ICRV. When this documentation receives the go-ahead, I'll figure out how best to merge it and add proctoring.

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 42]
```diff
@@ -0,0 +1,286 @@
...
+*****************************************
+Enable In-Course Identity Reverification
+*****************************************
+
+To enable in-course identity reverification for your course, follow these
+steps.
+
+#. In Studio, select **Settings**, and then select **Advanced Settings**.
+#. In the **Advanced Module List** field, place your cursor between the
+   brackets (``[]``), and then add ``"edx-reverification-block"``. Make sure to
```
Comment (srpearce) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: For clarity, add a space between the brackets (`[ ]`) - in the rendered HTML the two together just look like a box.

File: en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst [hunk 43]
```diff
@@ -0,0 +1,286 @@
...
+.. image:: ../Images/SFD_Credit_ReqList.png
+ :width: 350
+ :alt: Learner's Progress page with a list of credit requirements below the
+     progress graph.
+
+For more information about the in-course identity reverification experience
+for the learner, see `In-Course Identity Reverification <http://edx-guide-for-students.readthedocs.org/en/latest/SFD_credit_courses/SFD_in_course_ID_reverification.html>`_.
+
+*************************************************
+Create an In-Course Reverification Checkpoint
```
Comment (srpearce) [inline en_us/course_authors/source/building_course/credit_courses/in_course_reverification.rst]: This section contains the original procedure for ICRV, which has been changed. You can safely delete it.

File: en_us/course_authors/source/building_course/credit_courses/credit_courses.rst [hunk 44]
```diff
@@ -0,0 +1,76 @@
...
+If your course offers academic credit, your edX program manager will complete
+the initial setup of your course, including setting the price of credit and
+enabling your course to offer credit. You must then complete the following
+steps.
+
+#. :ref:`Specify the minimum grade that a learner must earn to receive
+   academic credit in the course <Specify Minimum Credit Eligible Grade>`.
+
+#. :ref:`Determine which identity checks you want to use and add identity
+   checks to the course <Add Identity Checks>`. We recommend that you read the
```
Comment (srpearce) [inline en_us/course_authors/source/building_course/credit_courses/credit_courses.rst]: Since "Add identity checks to the course" is now its own step, remove "and add identity checks..." from this sentence.

**Comment by srpearce** [comment]
Just needs a couple of changes.

**Comment by catong** [comment]
@mhoeber I made one commit to remove the unused image that I mentioned; other than that, it's good to go from the proctoring doc POV.  :+1:
